### ISSUE TITLE
Cannot open secondary roots when using multiple vaults

### ISSUE THREAD

**Issue by PhilipWhiteside** [dendronhq/dendron#495]
**Describe the bug**
When using multiple vaults only the top vault in the dendron pane will open its root. Other roots open the first vaults root. 

**To Reproduce**
1. Create 2 vaults
2. Click on 1st vault root (see breadcrumb path is correct)
3. Close tab
4. Click on 2nd vault root (see breadcrumb path is NOT correct, it shows the 1st vault root)

**Expected behavior**
Open the root file for the selected vault. 

**Screenshots**

![Screenshot from 2021-02-20 09-39-58 380462334](https://user-images.githubusercontent.com/22427598/108591257-b3c43600-735f-11eb-9332-02513a51e37f.png)
![Screenshot from 2021-02-20 09-40-03 523127699](https://user-images.githubusercontent.com/22427598/108591256-b32b9f80-735f-11eb-91bb-9bdea464c5e5.png)

**Desktop (please complete the following information):**
 - OS: Fedora 33 (Workstation Edition)
 - Browser: VSCodium 1.53.2, Electron 11.2.1, Chrome 87.0.4280.141, Node.js 12.18.3, V8 8.7.220.31-electron.0, OS Linux x64 5.10.15.200.fc33.x86_64
 - Version: Dendron v0.29.0

**Dendron Log file**
Please attach the output of `>Dev:Dendron: Open Logs` here

(This shows as `Dendron:Dev: Open Logs`)

```json
{"level":30,"time":1613810124934,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"Logger:configure","msg":"exit","log_level":"info"}
{"level":30,"time":1613810125010,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"_activate","stage":"dev","isDebug":false,"logPath":"/home/pwhitesi/.config/VSCodium/logs/20210220T083515/exthost1/dendron.dendron","logLevel":"info","extensionPath":"/home/pwhitesi/.vscode-oss/extensions/dendron.dendron-0.29.0","extensionUri":{"$mid":1,"fsPath":"/home/pwhitesi/.vscode-oss/extensions/dendron.dendron-0.29.0","path":"/home/pwhitesi/.vscode-oss/extensions/dendron.dendron-0.29.0","scheme":"file"},"storagePath":"/home/pwhitesi/.config/VSCodium/User/workspaceStorage/31064ccdb91541926dfa49d750726cfd/dendron.dendron","workspaceFile":{"$mid":1,"fsPath":"/home/pwhitesi/Documents/Notebook/dendron.code-workspace","external":"file:///home/pwhitesi/Documents/Notebook/dendron.code-workspace","path":"/home/pwhitesi/Documents/Notebook/dendron.code-workspace","scheme":"file"},"workspaceFolders":[{"uri":{"$mid":1,"fsPath":"/home/pwhitesi/Documents/Notebook/TestNewVault","external":"file:///home/pwhitesi/Documents/Notebook/TestNewVault","path":"/home/pwhitesi/Documents/Notebook/TestNewVault","scheme":"file"},"name":"TestNewVault","index":0},{"uri":{"$mid":1,"fsPath":"/home/pwhitesi/Documents/Notebook/Customer","external":"file:///home/pwhitesi/Documents/Notebook/Customer","path":"/home/pwhitesi/Documents/Notebook/Customer","scheme":"file"},"name":"Customer","index":1},{"uri":{"$mid":1,"fsPath":"/home/pwhitesi/Documents/Notebook/Notes","external":"file:///home/pwhitesi/Documents/Notebook/Notes","path":"/home/pwhitesi/Documents/Notebook/Notes","scheme":"file"},"name":"Notes","index":2}]}
{"level":30,"time":1613810125015,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"DendronWorkspace","msg":"initialized"}
{"level":30,"time":1613810125018,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"_activate","config":{"version":1,"vaults":[{"fsPath":"TestNewVault"},{"fsPath":"Customer"},{"fsPath":"Notes"}],"useFMTitle":true,"site":{"copyAssets":true,"siteHierarchies":["root"],"siteRootDir":"docs","usePrettyRefs":true,"title":"Dendron","duplicateNoteBehavior":{"action":"useVault","payload":["Personal","Notes","Customer","TestVaultForMultiRoot","TestNewVault"]}}},"configMigrated":false,"msg":"read dendron config"}
{"level":30,"time":1613810125194,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"_activate","wsConfig":{"folders":[{"path":"TestNewVault"},{"path":"Customer"},{"path":"Notes"}],"settings":{"dendron.rootDir":".","files.autoSave":"onFocusChange","pasteImage.path":"${currentFileDir}/assets/images","pasteImage.prefix":"/","markdown-preview-enhanced.enableWikiLinkSyntax":true,"markdown-preview-enhanced.wikiLinkFileExtension":".md","editor.snippetSuggestions":"inline","editor.suggest.snippetsPreventQuickSuggestions":false,"editor.suggest.showSnippets":true,"editor.tabCompletion":"on"},"extensions":{"recommendations":["dendron.dendron-paste-image","equinusocio.vsc-material-theme","dendron.dendron-markdown-shortcuts","dendron.dendron-markdown-preview-enhanced","dendron.dendron-markdown-links","github.github-vscode-theme"],"unwantedRecommendations":["dendron.dendron-markdown-notes","shd101wyy.markdown-preview-enhanced","kortina.vscode-markdown-notes","mushan.vscode-paste-image"]}},"wsConfigMigrated":{},"msg":"read wsConfig"}
{"level":30,"time":1613810125333,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"_activate","installedGlobalVersion":"0.29.0","migratedGlobalVersion":"0.29.0","previousGlobalVersion":"0.0.0","previousWsVersion":"0.29.0","platform":"linux","extensions":[{"id":"dendron.dendron-paste-image","version":"1.0.4","active":false},{"id":"equinusocio.vsc-material-theme","version":"33.1.2","active":false},{"id":"dendron.dendron-markdown-shortcuts","version":"0.12.1","active":true},{"id":"dendron.dendron-markdown-preview-enhanced","version":"0.10.23","active":false},{"id":"dendron.dendron-markdown-links","version":"0.6.19","active":false},{"id":"github.github-vscode-theme","version":"1.1.5","active":false}],"vaults":[{"fsPath":"TestNewVault"},{"fsPath":"Customer"},{"fsPath":"Notes"}]}
{"level":30,"time":1613810125336,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"startServer","logLevel":"info"}
{"level":30,"time":1613810125349,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"_activate","msg":"post-start-server","port":44903}
{"level":30,"time":1613810125354,"pid":139812,"hostname":"x1carbon","name":"logger","ctx":"FileStorageV2","wsRoot":"/home/pwhitesi/Documents/Notebook","vaultsv3":[{"fsPath":"TestNewVault"},{"fsPath":"Customer"},{"fsPath":"Notes"}],"level":"info"}
{"level":30,"time":1613810125355,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"ReloadIndex.execute","msg":"enter"}
{"level":30,"time":1613810128281,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"ReloadIndex.execute","msg":"exit"}
{"level":30,"time":1613810128281,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"reloadWorkspace","msg":"post-ws.reloadWorkspace"}
{"level":30,"time":1613810128283,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"reloadWorkspace","msg":"exit"}
{"level":30,"time":1613810128283,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"postReloadWorkspace","msg":"no previous global version"}
{"level":30,"time":1613810128283,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"postReloadWorkspace","msg":"exit"}
{"level":30,"time":1613810128284,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"TreeView:getChildren","msg":"reconstructing tree"}
{"level":30,"time":1613810128333,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"setupViews","msg":"init:backlinks"}
{"level":30,"time":1613810128335,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"activateWorkspace","stage":"dev","msg":"enter"}
{"level":30,"time":1613810128340,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"h:run","msg":"pre-execute"}
{"level":30,"time":1613810128340,"pid":139812,"hostname":"x1carbon","name":"UpgradeSettingsCommand","ctx":"Upgrade:execute"}
{"level":30,"time":1613810128345,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"toggleViews","msg":"views enabled: true"}
{"level":30,"time":1613810128345,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"_activate","msg":"fin startClient"}
{"level":30,"time":1613810128345,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"showWelcomeOrWhatsNew","version":"0.29.0","previousVersion":"0.29.0"}
{"level":30,"time":1613810128345,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"showWelcomeOrWhatsNew","msg":"not first time install"}
{"level":30,"time":1613810129171,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"TreeView:getChildren","msg":"reconstructing tree"}
{"level":30,"time":1613810129243,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"WorkspaceConfig:update","vscodeDir":"/home/pwhitesi/Documents/Notebook/TestNewVault/.vscode","snippetChanges":{}}
{"level":30,"time":1613810129243,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"Upgrade:execute","newConfig":{"extensions":{},"settings":{"add":{},"errors":{}},"snippetChanges":{}}}
{"level":30,"time":1613810129243,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"Upgrade:execute","badExtensions":[]}
{"level":30,"time":1613810129244,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"h:run","msg":"post-execute"}
{"level":30,"time":1613810129244,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"postReloadWorkspace","msg":"postUpgrade: new wsVersion"}
{"level":30,"time":1613810133375,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"pre-execute"}
{"level":30,"time":1613810133376,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"Customer"}},"msg":"enter"}
{"level":30,"time":1613810134326,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","msg":"enter","uri":{"$mid":1,"fsPath":"/home/pwhitesi/Documents/Notebook/TestNewVault/root.md","external":"file:///home/pwhitesi/Documents/Notebook/TestNewVault/root.md","path":"/home/pwhitesi/Documents/Notebook/TestNewVault/root.md","scheme":"file"}}
{"level":30,"time":1613810141373,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"Customer"}},"msg":"exit"}
{"level":30,"time":1613810141391,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"post-execute"}
{"level":30,"time":1613810141849,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"pre-execute"}
{"level":30,"time":1613810141849,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"Notes"}},"msg":"enter"}
{"level":30,"time":1613810141850,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"pre-execute"}
{"level":30,"time":1613810141850,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"TestNewVault"}},"msg":"enter"}
{"level":30,"time":1613810142202,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"Notes"}},"msg":"exit"}
{"level":30,"time":1613810142202,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"TestNewVault"}},"msg":"exit"}
{"level":30,"time":1613810142202,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"post-execute"}
{"level":30,"time":1613810142203,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"post-execute"}
{"level":30,"time":1613810143720,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"pre-execute"}
{"level":30,"time":1613810143720,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"Notes"}},"msg":"enter"}
{"level":30,"time":1613810143765,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"Notes"}},"msg":"exit"}
{"level":30,"time":1613810143765,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"post-execute"}
{"level":30,"time":1613810144594,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"pre-execute"}
{"level":30,"time":1613810144594,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"Customer"}},"msg":"enter"}
{"level":30,"time":1613810144644,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"Customer"}},"msg":"exit"}
{"level":30,"time":1613810144648,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"post-execute"}
{"level":30,"time":1613810145464,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"pre-execute"}
{"level":30,"time":1613810145464,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"TestNewVault"}},"msg":"enter"}
{"level":30,"time":1613810145519,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"GotoNoteCommand","opts":{"qs":"root","mode":"note","vault":{"fsPath":"TestNewVault"}},"msg":"exit"}
{"level":30,"time":1613810145519,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"d:run","msg":"post-execute"}
{"level":30,"time":1613810149053,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"c:run","msg":"pre-execute"}
{"level":30,"time":1613810149053,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"execute","opts":{}}
{"level":30,"time":1613810149094,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"c:run","msg":"post-execute"}
{"level":30,"time":1613810149354,"pid":139812,"hostname":"x1carbon","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","msg":"enter","uri":{"$mid":1,"fsPath":"/home/pwhitesi/.config/VSCodium/logs/20210220T083515/exthost1/dendron.dendron/dendron.log","external":"file:///home/pwhitesi/.config/VSCodium/logs/20210220T083515/exthost1/dendron.dendron/dendron.log","path":"/home/pwhitesi/.config/VSCodium/logs/20210220T083515/exthost1/dendron.dendron/dendron.log","scheme":"file"}}
```

**Comment by kevinslin** [comment]
Good catch! 

This should be a one line fix. Going to leave this up as a good first issue and fix this after a week if no one claims it. 

By the way, if you are on our [discord](https://discord.gg/AE3NRw9), feel free to ping me your handle since this earns you the [bug catcher role](https://www.kevinslin.com/notes/7c00d606-7b75-4d28-b563-d75f33f8e0d7.html) 😇

**Comment by PhilipWhiteside** [comment]
Hey :-) No I'm not, but I will join and let you know :+1: I'm new to Dendron so building up my usage

**Comment by kevinslin** [comment]
This has been fixed in 0.31 :)
